client_lib/test/unit/client/commands/repo/test_cudl.py

```diff
@@ -517,6 +517,36 @@ def test_summary(self):
         self.assertEqual(self.recorder.lines[0], 'abcdef  ABCDEF\n')
         self.assertEqual(self.recorder.lines[1], 'xyz     XYZ\n')
 
+    def test_summary_empty_display_name(self):
+        """
+            Test output of repo summary if display_name of one of the repos is not specified.
```

Comment:
There is no need for additional spaces at the beginning (take a look at other docstrings).